maven/bnd-maven-plugin/README.md

```diff
@@ -167,17 +148,15 @@ The plugin has 3 distinct usage scenarios:
    </plugin>
    ```
 
-3. The third case is best explained by building a _war_ file rather than a jar. The output of the plugin is effectively re-directed to a different location, in this case to the `maven-war-plugin` assembly directory:
+3. The third case is when the project packaging is `war`, for instance when the plugin is paired with the `maven-war-plugin` instead of the `maven-jar-plugin`. The output of the plugin re-directed to the default assembly directory of the `maven-war-plugin`, which is `${project.build.directory}/${project.build.finalName}`:
 
    ```xml
    <plugin>
        <groupId>biz.aQute.bnd</groupId>
        <artifactId>bnd-maven-plugin</artifactId>
        <configuration>
-           <outputDir>${project.build.directory}/${project.build.finalName}</outputDir>
            <bnd><![CDATA[
                Web-ContextPath: /${project.build.finalName}
```

Comment:
It is wrong to show the `<bnd>` example lines having leading spaces. Bnd will interpret this as a line continuation.

Reply:
it works perfectly in practice!

Reply:
do you really think this needs to be changed? It's really ugly in the POMs when you do that... meanwhile there is no evidence that it causes any problem.